types/lscache/index.d.ts

```diff
@@ -1,6 +1,7 @@
-// Type definitions for lscache v1.0.7
+// Type definitions for lscache v1.3.0
 // Project: https://github.com/pamelafox/lscache
 // Definitions by: Chris Martinez <https://github.com/Chris-Martinezz>
+//               : c-py <https://github.com/c-py>
```

Comment:
I don't think this is the accepted format for this field. Check something like react with lots of authors for an example.